builtin/aws/alb/releaser.go

```diff
@@ -51,6 +51,12 @@ func (r *Releaser) Release(
 		lbName = "waypoint-" + src.App
 	}
 
+	// We have to clamp at a length of 32 because the Name field to 
+	// CreateLoadBalancer requires that the name is 32 characters or less.
+	if len(lbName) > 32 {
+		lbName = lbName[:32]
```

Comment:
Maybe `log.Debug` here that we had to clamp the LB name due to the character restriction, with the original one in the message?

Reply:
Good idea! We do this clamping in several places -- should I add debugs to all of those in separate PRs or just in this one you think?

Reply:
Hrm, I don't mind it in here if they're all grouped into the same commit! I think that makes sense.